Ruby-Assignment-2-LoopingsAndLiterals/Question-1/solution2.rb

```diff
@@ -0,0 +1,31 @@
+
+TOTAL_CASES = gets.chomp.to_i
```

Comment:
please don't use the variable name in capital letters as in ruby it means it is a constant. 

Have a look http://rubylearning.com/satishtalim/ruby_constants.html